pipes/WDL/workflows/tasks/tasks_reports.wdl

```diff
@@ -15,6 +14,9 @@ task plot_coverage {
 
   Boolean? skip_mark_dupes=false
   Boolean? plot_only_non_duplicates=false
+  Boolean? bin_large_plots=false
+  
+  String sample_name = basename(basename(basename(reads_unmapped_bam, ".bam"), ".taxfilt"), ".clean")
```

Comment:
Great, thanks for this.

Reply:
Welcome. :)

Reply:
Ah, but sometimes the bam file is named `.clean.bam`.